spa/.yarnrc

```diff
@@ -0,0 +1 @@
+--ignore-engines true
```

Comment:
This is because one of the sub component in the latest react-script replies on node16. 
But the base node version on root folder relies on node 14 (dockerfile). 
I think we need to upgrade to node 16 later.